cli/pipeline/watch.go

```diff
@@ -106,8 +108,14 @@ func Watch(ctx context.Context, c kubernetes.Interface, watchPod *v1.Pod) (<-cha
 			select {
 			case <-time.After(watchPollInterval):
 				pod, err := c.CoreV1().Pods(watchPod.Namespace).Get(watchPod.Name, metav1.GetOptions{})
-				if err != nil {
+				if pod != nil {
```

Comment:
this is the recent pod status bug we encountered?

Reply:
Yup! You're right, it should have lived in a different PR but here it is :)